themes/janeswalk/js/components/pages/Dashboard/Walks.jsx

```diff
@@ -0,0 +1,84 @@
+
+import WalksFilter from './WalksFilter.jsx';
+import WalksMap from './WalksMap.jsx';
+import DashboardStore from './DashboardStore';
+
+//TODO: Walk common component found in <Itinerary/> and <WalkPage/>, Refactor to a single component or mixin (Post-PR)
+import Walk from './Walk.jsx';
+
+export default class Walks extends React.Component {
+  constructor(props, ...args) {
+    super(props, ...args);
+    this.state = {
+      activeFilters: props.activeFilters || [],
+      activeWalks: props.walks || [],
+      currentView: 'list',
+    };
+  }
+
+  //TODO: Hide Past Walks | Exports Spreadsheet (PR)
+  //TODO: Correct: This is doing an OR operation, not an AND operation (PR)
+  filterWalks() {
+    const {activeFilters} = this.state;
+    const {walks} = DashboardStore.getMyWalks();
```

Comment:
Not sure on the precedent of this naming: a dashboard (or rather, the profile page) can potentially be shown for other users. The dash store should show the walks for the profile's owner, not necessarily "my" walks.